tests/lib/at_custom_cmd/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(at_custom_cmd)
+
+FILE(GLOB app_sources src/*.c)
+target_sources(app PRIVATE ${app_sources})
+
+target_include_directories(app
+  PRIVATE
+  {ZEPHYR_NRF_MODULE_DIR}/nrf/include/modem/
```

Comment:
This becomes:
`<west-workspace>/nrf/nrf/include/modem/`

probably not your intention.
```suggestion
  {ZEPHYR_NRF_MODULE_DIR}/include/modem/
```
_

Hint: Next time, just click `Commit suggestion` to ensure you get the suggestion applied correctly :wink: 
![image](https://user-images.githubusercontent.com/9802583/195810358-cdc890f1-e518-4619-8a74-639e10ffdacc.png)

Reply:
I wonder if that `include` path is needed, as nothing seems to have failed in Jenkins.